chart/templates/job-db-migrate.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "mastodon.fullname" . }}-db-migrate
+  labels:
+    {{- include "mastodon.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
this may work better if it is pre-install,pre-upgrade

https://cloudnativedevopsblog.com/posts/database-migrations/

Reply:
`pre-install` hooks seem to run before anything is created, even PVCs.  `pre-upgrade` does seem to help, though (I'm sure I tried that last week; I must have had something else misconfigured)..